docs/announce-1.0.0.md

```diff
@@ -0,0 +1,54 @@
+  | ------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- |
+  | [NYC Taxi Demo](https://github.com/feathr-ai/feathr/blob/main/docs/samples/nyc_taxi_demo.ipynb)                                      | Quickstart notebook that showcases how to define, materialize, and register features with NYC taxi-fare prediction sample data.       | Azure Synapse, Databricks, Local Spark |
+  | [Databricks Quickstart NYC Taxi Demo](https://github.com/feathr-ai/feathr/blob/main/docs/samples/nyc_taxi_demo.ipynb)                | Quickstart Databricks notebook with NYC taxi-fare prediction sample data.                                                             | Databricks                             |
+  | [Feature Embedding](https://github.com/feathr-ai/feathr/blob/main/docs/samples/feature_embedding.ipynb)                              | Feathr UDF example showing how to define and use feature embedding with a pre-trained Transformer model and hotel review sample data. | Databricks                             |
+  | [Fraud Detection Demo](https://github.com/feathr-ai/feathr/blob/main/docs/samples/fraud_detection_demo.ipynb)                        | An example to demonstrate Feature Store using multiple data sources such as user account and transaction data.                        | Azure Synapse, Databricks, Local Spark |
+  | [Product Recommendation Demo](https://github.com/feathr-ai/feathr/blob/main/docs/samples/product_recommendation_demo_advanced.ipynb) | Feathr Feature Store example notebook with a product recommendation scenario                                                          | Azure Synapse, Databricks, Local Spark |
+
+- Web UI has been enhanced in following areas
+  - Support register feature
+  - Support delete feature
+  - Support display version
+  - Lineage has been added to top manu for quick access
+  You can try these new changes on Feathr live demo site at [Feathr Feature Store (feathr-registry-sql.azurewebsites.net)](https://feathr-registry-sql.azurewebsites.net/projects)](https://feathr-registry-sql.azurewebsites.net/)
+
+- Time pattern support in data source path.  [Input File for Feathr | Feathr (feathr-ai.github.io)](https://feathr-ai.github.io/feathr/how-to-guides/feathr-input-format.html#timepartitionpattern-for-input-files)
+
+- Feature names conflicts check and auto correction [Getting Offline Features using Feature Query | Feathr (feathr-ai.github.io)](https://feathr-ai.github.io/feathr/concepts/get-offline-features.html#feature-names-conflicts-check)
+
+- Composite key improvement , to be updated by @enya
+
+- Use SparkSQL as DataSource, to be updated by @yuqing
+
+For more information, please refer to the official Feathr documentation.
```

Comment:
Shall we apply a folder to maintain all docs related to release？

Reply:
Good suggestion, created a new release-announcements folder and moved doc